secboot/secboot_tpm.go

```diff
@@ -849,9 +826,9 @@ func tpmReleaseResourcesImpl(tpm *sb_tpm2.Connection, handle tpm2.Handle) error
 	return nil
 }
 
-// ReleasePCRResourceHandles releases any TPM resources associated with given
+// releasePCRResourceHandles releases any TPM resources associated with given
 // PCR handles.
-func ReleasePCRResourceHandles(handles ...uint32) error {
+func releasePCRResourceHandles(handles ...uint32) error {
```

Comment:
I may add this functionality to remove old NV indexes the secboot package, as it should handle the case where a handle it's told to undefine has already been undefined (again, related to use of the resource managed character device and the fact that the kernel can context switch access to the TPM between commands). If we execute `TPM2_NV_UndefineSpace` and it returns an error indicating that it doesn't exist, the function should filter out that error and treat it as success.

Reply:
A small nit about this function, although it existed before, is that this has nothing to do with PCRs. It's undefining NV indexes, so it seems a bit weird to have "PCR" in the function name.

Reply:
I will add a fixme.